paddle/phi/kernels/sparse/unary_grad_kernel.h

```diff
@@ -121,5 +121,22 @@ void ReshapeCsrGradKernel(const Context& dev_ctx,
                           const SparseCsrTensor& dout,
                           SparseCsrTensor* dx);
 
+template <typename T, typename Context>
+void SliceCooGradKernel(const Context& dev_ctx,
+                        const SparseCooTensor& x,
+                        const SparseCooTensor& out_grad,
+                        const phi::IntArray& axes_arr,
+                        const phi::IntArray& starts_arr,
+                        const phi::IntArray& ends_arr,
```

Comment:
Why use parameter name of `axes_arr`, `starts_arr` and `ends_arr`? No other kernel are not named like this, just using `axes`, `starts` and `ends` is better?

Reply:
same reason as above